src/app/rundown-view/components/scrollable-timeline/scrollable-timeline.component.ts

```diff
@@ -52,6 +54,19 @@ export class ScrollableTimelineComponent {
     )
   }
 
+  private checkForSpanningElements(parts: readonly Part[]): Part[] {
```

Comment:
Same comment for this as the other place you have this method.

Also, you already have this method... 
We should avoid duplicated code as much as possible.